worth2/main/generic/models.py

```diff
@@ -0,0 +1,55 @@
+from django import forms
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+from pagetree.models import PageBlock
+
+
+class BasePageBlock(models.Model):
+    """An abstract pageblock to be used for custom pageblocks."""
```

Comment:
Yeah definitely. I almost made a pull request for pagetree tonight but I figure I'll wait and see how it works in worth.